docs/book/v2/quick-start.md

```diff
@@ -0,0 +1,138 @@
+# Quick Start
+
+This quick start is intended to get developers familiar with the concepts of the
+laminas-di DiC.  Generally speaking, code is never as simple as it is inside this
+example, so working knowledge of the other sections of the manual is suggested.
+
+Assume, for a moment, the following application code. It already assumes that
+dependencies are injected, and so becomes a good candiate for a DiC.
+
+```php
+namespace MyLibrary
+{
+    class DbAdapter
+    {
+        protected $username = null;
+        protected $password = null;
+
+        public function __construct($username, $password)
+        {
+            $this->username = $username;
+            $this->password = $password;
+        }
+    }
+}
+
+namespace MyMovieApp
+{
+    class MovieFinder
+    {
+        protected $dbAdapter = null;
+
+        public function __construct(\MyLibrary\DbAdapter $dbAdapter)
+        {
+            $this->dbAdapter = $dbAdapter;
+        }
+    }
+
+    class MovieLister
+    {
+        protected $movieFinder = null;
+
+        public function __construct(MovieFinder $movieFinder)
+        {
+            $this->movieFinder = $movieFinder;
+        }
+    }
+}
+```
+
+With the above code, you find yourself writing the following to wire and utilize
+it:
+
+```php
+// $config object is assumed
+
+$dbAdapter = new MyLibrary\DbAdapter($config->username, $config->password);
+$movieFinder = new MyMovieApp\MovieFinder($dbAdapter);
+$movieLister = new MyMovieApp\MovieLister($movieFinder);
+foreach ($movieLister as $movie) {
+    // iterate and display $movie
+}
+```
+
+If you are doing this above wiring in each controller or view that wants to list
+movies, not only can this become repetitive and boring to write, but also
+unmaintainable if you want to swap out one of these dependencies on a wholesale
+scale.
+
+Since this example of code already practices good dependency injection using
+constructor injection, it is a great candidate for using laminas-di.
+
+The following demonstrates how to wire the above into a laminas-di container:
```

Comment:
A headline can be added here. For example: "Using a laminas-di Container"

Reply:
Oops, added the headline a paragraph too early. Should be fixed now.